sdk/core/src/main/java/com/klaviyo/core/KLog.kt

```diff
@@ -6,7 +6,26 @@ import com.klaviyo.core.config.Log
 import com.klaviyo.core.config.Log.Level
 import java.util.regex.Pattern
 
-open class KLog : Log {
+object KLog : Log {
```

Comment:
`KLog` doesn't need to be instantiated, it is essentially just a set of static methods anyway.